### PR TITLE
Feedback status

### PR DESCRIPTION
**Pull request by Buckram123**
@adairrr Thanks for the feedback, added comments on unique @feedback's
- `✅`: Clear, no clarification needed (Done/WIP)
- `// Comment`: Just comment, feel free to ignore if you agree with comment 
- `@uno-feedback`: Need more clarification

### CODE REVIEW

**Comment by adairrr** [comment]
Awesome thank you @Buckram123 !

File: src/bin/create_xion_icaa.rs [hunk 1]
```diff
@@ -80,18 +80,19 @@ fn icaa_demo() -> anyhow::Result<()> {
         .account_builder()
         .name("ICAA PL Test")
         // @feedback: this namespace method should note that remote namespaces will not work
+        // @uno-feedback: Which namespace method? You mean we should note that you can't have sub-account from different chain?
```
Comment (adairrr) [inline src/bin/create_xion_icaa.rs:83]: Hm. I believe this was with the thought that we’d have a renote_account method on the builder, and that when you add a namespace there it won’t actually work. However given the remote account builder being separate it should not be an issue. 

Unless I’m failing to remember…

File: src/lib.rs [hunk 2]
```diff
@@ -23,12 +23,13 @@ pub const JUNO_1: ChainInfo = ChainInfo {
 pub const IBC_CLIENT_ID: &str = "abstract:ibc-client";
 
 // @feedback: it would be really nice to be able to query a module directly from the account
+// @uno-feedback: Why not use abstract-client `Application` helper?
```
Comment (adairrr) [inline src/lib.rs:26]: Didn’t know we could use it for the ibc-client at this point. Would be great to have examples on using it with it on the docs to demonstrate its use.

File: src/bin/create_xion_icaa.rs (hunk 1 above)
Comment (Buckram123) [reply src/bin/create_xion_icaa.rs:83]: Got it, thanks! That brings up a good point not sure why we allow claiming namespace?

File: src/lib.rs (hunk 2 above)
Comment (Buckram123) [reply src/lib.rs:26]: I think it's still worth adding `query_module(module_id, &msg)` on `Account`, do you agree? It's possible that someone else abstract module(most likely standalone) won't have required traits to get `Application` object

**Pull request closed without merging**